README.md

```diff
@@ -11,11 +18,26 @@ Mod Engine 2 is a ground up rewrite of Mod Engine, a runtime code patching and i
 See the list below for information on games that currently have support in Mod Engine 2.
 
 - [x] Dark Souls 3
+- [x] Elden Ring
 - [ ] Dark Souls 2: SOTF
 - [ ] Dark Souls Remastered
 - [ ] Sekiro: Shadows Die Twice
 - [ ] Bloodborne
 
+## Get started guide
+
+1. Download the latest [release](https://github.com/soulsmods/ModEngine2/releases).
+2. Create a folder in the mod folder, I recommend the name of the mod for clarity.
```

Comment:
Do you mean: "Create a folder in the ~~mod~~ game folder"?

Reply:
No there is a mod folder included in the zip file you can download from the release section. That mod folder is what I mean here.